packages/dbml-core/src/parse/databaseGenerator.js

```diff
@@ -74,7 +74,7 @@ const createIndexes = (rawIndexes) => {
 const createTables = (rawTables, rawFields, rawIndexes, tableConstraints) => {
   return rawTables.map((rawTable) => {
     const { name, schemaName, note } = rawTable;
-    const key = `${schemaName}.${name}`;
+    const key = `${schemaName || ''}.${name}`;
```

Comment:
I think it should be changed to:
```javascript
const key = schemaName ? `${name}` : `${schemaName}.${name}`;
```

Reply:
Fixed in https://github.com/holistics/dbml/pull/602/commits/f0dc9d52059958f8e1042c98657b695762d8ef2c